lib/opensusebasetest.pm

```diff
@@ -197,10 +197,12 @@ sub set_standard_prompt {
 sub select_bootmenu_more {
     my ($self, $tag, $more) = @_;
 
-    assert_screen "inst-bootmenu", 15;
+    # do not waste time waiting when we already matched
+    assert_screen 'inst-bootmenu', 15 unless match_has_tag 'inst-bootmenu';
+    stop_grub_timeout;
```

Comment:
let's imagine that previous match was not 'inst-bootmenu' , in this case you could assert_screen for 15 seconds so there is a big chance that it's too late to stop something, right ?
Also I had impression that there some kind of agreement that we just disable grub timeout usually to not fight with such kind of problems ?

Reply:
Him, not really as a single assert screen should take less than a second, multiple ones as in before are a waste but the actual problem are only the check screens that came afterwards

Reply:
and second question - didn't we have disabled grub timeout so we don't need to stop something ?

Reply:
yes, we can. I will move it up